views/js/components/nav-bar-top.js

```diff
@@ -1,66 +1,81 @@
 import { renderLogin } from './login.js'
+import { logout } from './logout.js'
 
 // Render top navigation bar
 export const renderNavBar = () => {
-    const navBar = document.getElementById('nav-bar');
-    navBar.innerHTML = '';
-    
-    const h1 = document.createElement('h1');
-    h1.id = 'logo';
-    h1.textContent = 'TRIPT';
-    navBar.appendChild(h1);
+    axios.get('/user/session')
+    .then(dbRes => {
+        if (dbRes) {
```

Comment:
Here and in the catch block we are repeating code for creating navbar elements; maybe from the API we return a response in object and based on which key is present (logged in or not), we create the elements with the pertinent details?

response object - 
(if logged out) - {Log In : true}
(if logged in) - {Log Out : true}

const button = ... (create element)
button.textContent = Object.keys(response)[0]

Let me know what you think Chris!